instance/templates/instance/ansible/vars.yml

```diff
@@ -2,7 +2,7 @@
 COMMON_HOSTNAME: '{{ instance.domain }}'
 
 # edxapp
-EDXAPP_PLATFORM_NAME: "{{ instance.name }}"
+EDXAPP_PLATFORM_NAME: "{{ instance.instance.name }}"
```

Comment:
@bradenmacdonald This will become `appserver.instance.name` once we address [this TODO item](https://github.com/open-craft/opencraft/blob/provision-while-running/instance/models/openedx_appserver.py#L186), right? I think it would be worth doing that before merging this PR; `instance.instance.name` is a little confusing.